express/validations/users.js

```diff
@@ -19,6 +19,15 @@ const validations = {
 				),
 			filterValue: Joi.string()
 				.regex(common.filterValueRegex),
+			pageNumber: Joi.number()
+				.integer()
+				.positive(),
+			instancesNumber: Joi.number()
+				.valid(
+					5,
+					7,
+					10
```

Comment:
Also, there is no need to hardcode those options, you can simple set min, max and default values and allow to pass any limit number in that range